server/src/assembly/resources/conf/logback.xml

```diff
@@ -192,7 +192,7 @@
         <appender-ref ref="FILEDEBUG"/>
         <appender-ref ref="FILEWARN"/>
         <appender-ref ref="FILEERROR"/>
-        <appender-ref ref="FILEALL"/>
```

Comment:
The issue is to resolve the problem that the log_info.log do not contains the INFO level log,
log_all.log contains logs which  level is greater than or equal to INFO level,  please do not remove the 'FILEALL' log.